pkg/metrics/metrics.go

```diff
@@ -159,6 +159,108 @@ var (
 		Name:      "policy_l7_received_total",
 		Help:      "Number of total L7 received requests/responses",
 	})
+
+	// Status
+
+	// ControllersFailing is a count of failing controllers
+	ControllersFailing = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "controllers_failing_total",
+		Help:      "Number of faling controllers",
+	})
+
+	// IPv4Addresses is a count of allocated IPv4 addresses in the cluster
+	IPv4Addresses = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "ipv4_addresses_total",
+		Help:      "Number of allocated IPv4 addresses",
+	})
+
+	// IPv6Addresses is a count of allocated IPv6 addresses in the cluster
+	IPv6Addresses = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
You could consider combining these IPv4/6 metrics and then just adding a label `family="ipv4"`, `family="ipv6"` or similar.